elements/lisk-transactions/src/utils/sort.ts

```diff
@@ -15,22 +15,17 @@
 
 import { AccountUnlocking } from '../types';
 
-export const sortKeysAscending = (publicKeys: string[]): string[] =>
-	publicKeys.sort((publicKeyA, publicKeyB) => {
-		if (publicKeyA > publicKeyB) {
-			return 1;
-		}
-		if (publicKeyA < publicKeyB) {
-			return -1;
-		}
-
-		return 0;
-	});
+export const sortKeysAscending = (
+	publicKeys: Array<Readonly<Buffer>>,
+): Array<Readonly<Buffer>> =>
+	publicKeys.sort((publicKeyA, publicKeyB) =>
+		(publicKeyA as Buffer).compare(publicKeyB as Buffer),
```

Comment:
if above, i think there is no need to cast them?

Reply:
keys are `Array<Readonly<Buffer>>` so it wont work as i have to cast them and send, so i made this change to support